.github/workflows/deploy.yml

```diff
@@ -137,3 +138,18 @@ jobs:
           platforms: linux/amd64,linux/arm64/v8
           tags: ${{ steps.distroless-meta.outputs.tags }}
           push: true
+
+      - name: Build local distroless image, which is needed to run Snyk
+        if: fromJSON(env.RUN_SNYK)
```

Comment:
Otherwise `env.RUN_SNYK` isn't a boolean. More elegant suggestions are quite welcome.

Reply:
Maybe you can get this working using [an expanded matrix configuration](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations).  Something like:

```
  matrix:
    platform:
        - kinesis
        - sqs
        - pubsub
        - kafka
        - nsq
        - stdout
        - rabbitmq
    include:
      - platform: kinesis
        run_snyk: true
      - platform: pubsub
        run_snyk: true
```

Reply:
Yep, this seems to work nicely. Thanks for the suggestion.